src/util.ts

```diff
@@ -954,3 +954,14 @@ export function runGodoc(cwd: string, packagePath: string, receiver: string, sym
 		});
 	});
 }
+
+export function getTimeoutConfiguration(operationType: string, execFileOptions: {[key: string]: any}) {
+	let editor = vscode.window.activeTextEditor;
+	if (editor) {
+		const goConfig = vscode.workspace.getConfiguration('go', editor.document ? editor.document.uri : null);
+		const execTimeout: { [key: string]: any } = goConfig.get('operationTimeout');
+		if (execTimeout.hasOwnProperty(operationType)) {
+			execFileOptions['timeout'] = execTimeout[operationType];
+		}
+	}
```

Comment:
I prefer returning the timeout from this function instead of updating the options in place.
This would make the code more readable from the different files than first forming the options variable and then passing it to a function to be updated.

Also, we have a few places where we use `spawn` instead of `execFile` in which case the use of word `execFile` here can be misleading.